packages/host/app/components/operator-mode/card-url-bar.gts

```diff
@@ -38,11 +38,20 @@ export default class CardURLBar extends Component<Signature> {
       ...attributes
     >
       <div class='realm-info' data-test-card-url-bar-realm-info>
-        <div class='realm-icon'>
-          <img src={{this.realmIcon}} alt='realm-icon' />
-        </div>
-        <span>in
-          {{if this.realmName this.realmName 'Unknown Workspace'}}</span>
+        <RealmInfoProvider @realmURL={{@realmURL}}>
+          <:ready as |realmInfo|>
+            <div class='realm-icon'>
+              <img src={{realmInfo.iconURL}} alt='realm-icon' />
+            </div>
+            <span>in {{realmInfo.name}}</span>
+          </:ready>
+          <:error>
+            <div class='realm-icon'>
+              <img src='' alt='realm-icon' />
```

Comment:
Is img with a blank src valid html?

Reply:
Replaced it [254f2f2](https://github.com/cardstack/boxel/pull/700/commits/254f2f2c9a8b1ea6f0b6e9c2d900a42766f1ce2e). It will look like this.
![Screenshot 2023-10-03 at 19 26 43](https://github.com/cardstack/boxel/assets/12637010/a83135d8-925c-4c22-a968-01cdd8b16f07)